src/layouts/stressPatientDashboard/stressPatientDashboard.js

```diff
@@ -0,0 +1,18 @@
+import './stressPatientDashboard.css'
+import Header from '../../components/header/header.js'
+import Title from '../../components/title/titleGraphPatient.js'
+import StessComment from '../../components/stressComment/stressComment.js'
+import Graph from '../../components/graph/graph'
+
+function StressPatientDashboard() {
+  return (
+    <div className="body">
```

Comment:
I think we're currently not using this class. If we do, I would prefer a different naming. 

```suggestion
    <div>
```